synapse/federation/federation_server.py

```diff
@@ -1399,7 +1399,7 @@ async def on_edu(self, edu_type: str, origin: str, content: dict) -> None:
         # Check if we have a handler on this instance
         handler = self.edu_handlers.get(edu_type)
         if handler:
-            with start_active_span_from_edu(content, "handle_edu"):
+            with start_active_span_from_edu("handle_edu", edu_content=content):
```

Comment:
We don't seem to have the same homeserver whitelist (`whitelisted_homeserver`) checks around this like other places we worry about stuff like this. I assume we do want them here?

But [can we just rip all of this out](https://github.com/matrix-org/synapse/pull/13400#discussion_r934978416)?